students/Roy_Tate/lesson03/mailroom.py

```diff
@@ -0,0 +1,110 @@
+# author/student: Roy Tate (githubtater)
+donor_list = [
+    ['Bill', 400.254, 100, 200.232, 300],
+    ['Mark', 250, 450.355],
+    ['Jeff', 1000, 300000.23, 2000],
+    ['Female CEO 1', 1030, 20030.171, 30, 30],
+    ['Female CEO 2', 1030, 20030.355, 30, 30],
+]
+
+
+def donor_names():
+    names = []
+    for donor in donor_list:
+        names.append(donor[0])
+    return names
+
+def print_list():
+    print('{:*^50}'.format('DONOR LIST'))
+    for donor in donor_names():
+        print(donor)
+
+
+def add_donation(donor_name, new_donation):
+    for index, donor in enumerate(donor_list):
+        if donor[0] == donor_name:
+            donor_list[index].append(new_donation)
```

Comment:
since you get a reference to the donor item in the list, you don't need to append by index, basically following will work as well:
```
for donor in donor_list:
     if donor[0] == donor_name:
        donor.append(new_donation)
```
I would also call this function `add_new_donor` or combine to have functionality to add new donor and to add to existing donor